fuel-core/src/service/modules.rs

```diff
@@ -56,9 +59,15 @@ pub async fn start_modules(config: &Config, database: &Database) -> Result<Modul
                 .unwrap(),
         );
 
+    // Meant to simulate p2p's channels which hook in to communicate with txpool
+    let (network_sender, _) = mpsc::channel(100);
```

Comment:
Without any network_receiver, this queue will fill up and may stop the txpool from working once it's reached 100 txs.

We likely need to spawn a task to drain this queue whenever txs are added.